src/main/features/first-contact/views/start.njk

```diff
@@ -36,11 +38,9 @@
         <h2 class="heading-medium" id="subsection-title">{{ t('Support') }}</h2>
         <nav role="navigation" aria-labelledby="subsection-title">
           <ul class="font-xsmall">
-            <li><a href="https://www.gov.uk/make-court-claim-for-money/court-fees">{{ t('Money claim fees') }}</a></li>
-            <li><a href="/other-options">{{ t('Other options') }}</a></li>
-            <li><a class="external" rel="external"
-                   href="https://courttribunalfinder.service.gov.uk/courts/county-court-business-centre-ccbc"
-                   target="_blank">{{ t('Contact us') }}</a></li>
+            <li>{{ externalLink(t('Money claim fees'), 'https://www.gov.uk/make-court-claim-for-money/court-fees') }}</li>
+            <li>{{ internalLink(t('Other options'), '/other-options') }}</li>
```

Comment:
this should also be in a Path file. I'm wondering if we should put the external ones in one common Paths file as well? might be best to keep that for another story though

Reply:
The `Other options` link goes to a missing page. Waiting for feedback from UX guys on it.

Reply:
This version of the first-contact start page is only for local use - the live system will point users to a GOV.UK page instead. Removed the dead link.